src/activities/components/ActivityList.vue

```diff
@@ -334,9 +334,50 @@ export default {
       icsDialog: false,
       numDisplayed: NUM_ACTIVITIES_PER_LOAD,
       types: [],
+      activityListTypeFilter: this.filterActivityTypes.slice(),
     }
   },
   computed: {
+    activityTypes () {
+      const archivedTypeList = []
+      const archivedTypes = {}
+
+      for (const type of this.activityListTypeFilter) {
+        // populate a list of all activity types locally to avoid side effects
+        archivedTypeList.push(type)
```

Comment:
Seems you're just cloning a list here, which you can do in a simpler/cleaner way (off the top of my head):

```js
const newList = [...oldList]
// or
const newList = oldList.slice()
```